src/services/feed.ts

```diff
@@ -0,0 +1,71 @@
+import * as Feed from '../types/feed';
+import { Service } from '../types/reyah';
+import * as Status from '../types/status';
+import { reyahServiceRequest } from '../core/core';
+import newServiceStatus from '../constructor/status';
+import { dispatchError } from '../core/errors';
+import { Pagination } from '../types/pagination';
+import { newFeedElements } from '../constructor/feed';
+
+export class FeedService implements Service {
+    readonly subpath = '/feed';
+
+    /**
+     * Remote service status
+     * @return whether the service is alive or not
+     */
+    public async alive(): Promise<Status.ServiceStatus> {
```

Comment:
Waiting for the addition of a healthcheck method

Reply:
I think it should be good now ?